woocommerce/woocommerce-ios
# Blaze: Update dark mode color for intro view and campaign creation form

Closes: #11735 


## Description

This PR updates the colors on the intro view and campaign creation form to match the design (mBhyoV7OPhFnOugxLy6SMS-fi-2506-59795).

## Testing instructions

- Log in to a store that's eligible for Blaze and doesn't have any campaign created yet.
- Tap on the Promote button on the Blaze section on the dashboard screen.
- Notice that the intro screen has colors matching the design in both light and dark modes.
- Tap Create Your Campaign, notice that the creation form is pushed with the tab bar hidden.
- Notice that the colors on the creation form match the design in both light and dark modes.

## Screenshots


\ | Light mode | Dark mode 
--- | --- | ---
Intro view | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/4b7a7ccc-d0f6-4871-8ee0-68a10f9a948e" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/8aad01c4-a866-4f95-823b-29fcfadcab00" width=320 />
Creation form | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/bc0cccad-b515-4fa5-b1d9-661307d846c1" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/2c8d1fb1-158a-4b60-8f6a-b5fa2097c74e" width=320 /> 




---
